docs/usage/configuration-options.md

```diff
@@ -1893,17 +1893,18 @@ Just like the earlier `matchPackagePatterns` example, the above will configure `
 
 ### matchPaths
 
-Renovate will match `matchPaths` against both a partial string match or a minimatch glob pattern.
-If you want to avoid the partial string matching so that only glob matching is performed, wrap your string in `+(...)` like so:
+Renovate finds the file(s) listed in `matchPaths` with a minimatch glob pattern.
 
-```
-  "matchPaths": ["+(package.json)"],
+For example the following would match any `package.json`, including files like `backend/package.json`:
+
+```json
+  "matchPaths": ["package.json"],
```

Comment:
```suggestion
{
  "packageRules": [
    {
      "description": "Find package.json files and do INSERT EXPLANATION",
      "matchPaths": ["package.json"],
    }
  ]
}
```

Reply:
I'd say this is mostly used to group dependencies somehow, does this help?

```suggestion
"matchPaths": ["package.json"],
{
  "packageRules": [
    {
      "description": "Group dependencies from package.json files",
      "matchPaths": ["**/package.json"],
      "groupName": "All package.json changes "
    }
  ]
}
```

Reply:
(I also changed `package.json` to `**/package.json` to make clear that this needs to be a pattern)

Reply:
```json